src-ghc/Pact/Server/PactService.hs

```diff
@@ -91,18 +93,67 @@ runPayload c@Command{..} = do
         _ -> return $ jsonResult mode (cmdToRequestKey c) $ CommandError "Private" Nothing
     Nothing -> runRpc _pPayload
 
-
-
 applyExec :: RequestKey -> ExecMsg ParsedCode -> Command a -> CommandM p CommandResult
 applyExec rk (ExecMsg parsedCode edata) Command{..} = do
   CommandEnv {..} <- ask
   when (null (_pcExps parsedCode)) $ throwCmdEx "No expressions found"
-  (CommandState refStore) <- liftIO $ readMVar _ceState
-  let evalEnv = setupEvalEnv _ceDbEnv _ceEntity _ceMode
-                (MsgData (userSigsToPactKeySet _cmdSigs) edata Nothing _cmdHash) refStore
-  pr <- liftIO $ evalExec evalEnv parsedCode
-  void $ liftIO $ swapMVar _ceState $ CommandState (erRefStore pr)
-  return $ jsonResult _ceMode rk $ CommandSuccess (last (erOutput pr))
-
-applyContinuation :: ContMsg -> [UserSig] -> CommandM p CommandResult
-applyContinuation _ _ = throwCmdEx "Continuation not supported"
+  CommandState {..} <- liftIO $ readMVar _ceState
+  let sigs = userSigsToPactKeySet _cmdSigs
```

Comment:
should pass `sigs` to `handleYield` no?

Reply:
When I try passing in sigs to handleYield, it gives me this error:
```
src-ghc/Pact/Server/PactService.hs:113:37-45: error: …
    Not in scope: type constructor or class ‘PublicKey’
    Perhaps you want to remove ‘PublicKey’
    from the explicit hiding list in the import of ‘Pact.Types.Runtime’
    (/Users/linda.ortega.cordoves/pact/src-ghc/Pact/Server/PactService.hs:30:1-44).
Compilation failed.
```
based on this line in the same file: ```import Pact.Types.Runtime hiding (PublicKey)```
Why is PublicKey being hidden?